contracts/axelar-gateway/src/test_gateway.rs

```diff
@@ -10,6 +10,7 @@ use axelar_soroban_interfaces::types::Message;
 use axelar_soroban_std::{assert_contract_err, assert_invocation, assert_last_emitted_event};
 use soroban_sdk::testutils::BytesN as _;
 
+use soroban_sdk::Symbol;
 use soroban_sdk::{
     bytes, symbol_short,
```

Comment:
unused import: `symbol_short`

Reply:
hmm, not sure why actions didn't complain. maybe take a look later to see if we need to enable certain lints